CRM/Utils/System/Drupal8.php

```diff
@@ -64,19 +65,29 @@ public function createUser(&$params, $mail) {
     if ($user_register_conf != 'visitors' && !$user->hasPermission('administer users')) {
       $account->block();
     }
+    elseif (!$verify_mail_conf) {
+      $account->activate();
```

Comment:
The default value of the 'status' field is 0 (or blocked), so unless some other module is altering that, we need to explicitly activate it here.